src/Adapter/Product/Image/CommandHandler/UpdateProductImageHandler.php

```diff
@@ -81,18 +84,46 @@ public function __construct(
      */
     public function handle(UpdateProductImageCommand $command): void
     {
+        $shopConstraint = $command->getShopConstraint();
+
+        if ($shopConstraint->getShopGroupId()) {
+            throw new InvalidShopConstraintException('Shop group constraint is not supported');
```

Comment:
```suggestion
            throw new InvalidShopConstraintException('Store group constraint is not supported');
```

Reply:
this one is raw exception message its not supposed to be shown directly to the end-user and doesn't need adjustment